garnett/utils.py

```diff
@@ -68,14 +69,28 @@ def get_current_blank_override() -> bool:
     return _ctx_force_blank.get(False)
 
 
+def get_safe_language(lang_code: str) -> Optional[Language]:
+    """Return language if language for lang code exists, otherwise none"""
+    try:
+        return Language.get(lang_code)
+    except langcodes.tag_parser.LanguageTagError:
+        return None
+
+
 def get_languages() -> List[Language]:
     langs = getattr(
         settings, "GARNETT_TRANSLATABLE_LANGUAGES", [get_default_language()]
     )
     if callable(langs):
         langs = langs()
     if type(langs) == list:
-        return [Language.get(lang) for lang in langs]
+        languages = []
+        for lang_code in langs:
+            if language := get_safe_language(lang_code):
+                languages.append(language)
+
+        return languages
```

Comment:
Could this break if it is empty?

Reply:
It should - garnett should fail with an ImproperlyConfigured error if languages is empty.